frontend/src/routes/guide/GuidePage.vue

```diff
@@ -0,0 +1,47 @@
+<script setup lang="ts">
+import { useRoute, useRouter, type RouteLocationNamedRaw } from 'vue-router';
+import NavBar from '../../components/NavBar.vue';
+import GuideBottomBar from './GuideBottomBar.vue';
+import { computed, type ComputedRef } from 'vue';
+import { appRoutes } from '@/main';
+const router = useRouter();
+const route = useRoute();
+const pageNr = computed(() => (route.params.nr ? route.params.nr : '1'));
+const bottomBarSubtitle = computed(() => `navod ${pageNr.value}/2`);
+const buttonTitle = computed(() => {
+  if (pageNr.value === '1') {
+    return 'Pokracovat';
+  } else {
+    return 'Prvni otazka';
+  }
+});
+const buttonRoute: ComputedRef<RouteLocationNamedRaw> = computed(() => {
+  if (pageNr.value === '1') {
```

Comment:
Přidej prosím ten guide jako 1+3 kroky, stejně jako na mobilu, jak jsme se dohodli, že to sjednotíme.